src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -1,34 +1,135 @@
 package seedu.address.logic.commands;
 
-import seedu.address.model.pet.Gender;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DOB;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GENDER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SPECIES;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.general.Command;
+import seedu.address.logic.commands.general.CommandResult;
+import seedu.address.logic.commands.general.exceptions.CommandException;
+import seedu.address.logic.commands.pet.EditPetCommand;
+import seedu.address.model.Model;
+import seedu.address.model.PetTracker;
+import seedu.address.model.pet.Gender;
+import seedu.address.model.pet.NameContainsKeywordsPredicate;
+import seedu.address.model.pet.Pet;
+import seedu.address.testutil.pet.EditPetDescriptorBuilder;
 
 /**
  * Contains helper methods for testing commands.
  */
 public class CommandTestUtil {
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
     public static final String VALID_NAME_COCO = "Coco";
-    public static final String VALID_NAME_GARFIELD = "Garfield Arbuckle";
-    public static final String VALID_PHONE_AMY = "11111111";
-    public static final String VALID_PHONE_BOB = "22222222";
+    public static final String VALID_NAME_GARFIELD = "Garfield";
     public static final Gender VALID_GENDER_COCO = Gender.FEMALE;
     public static final Gender VALID_GENDER_GARFIELD = Gender.MALE;
-    public static final String VALID_EMAIL_AMY = "amy@example.com";
-    public static final String VALID_EMAIL_BOB = "bob@example.com";
     public static final String VALID_DOB_COCO = "2-2-2015";
```

Comment:
update DOB format to be consistent with Dylan's PR. 2-2-2015 should change to 2/2/2015